packages/http/src/types.ts

```diff
@@ -277,7 +278,7 @@ export interface RouteResolutionOptions extends RouteOptions {
 }
 
 export interface RouteProducerResult {
-  segments: string[];
```

Comment:
This is a breaking api change, isn't it?  We should document

Reply:
well this is for the route producer thing that is used by rest library to get its own route producing wiht @autoRoute. 

All of this is completely undocumented, what do you think?